tutorial/server/axum/assets/js_index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>WebAuthn-rs Tutorial</title>
+	<script src="https://cdn.jsdelivr.net/npm/js-base64@3.7.4/base64.min.js"></script>
```

Comment:
As this pins a specific version of the package, can this use [subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) and `crossorigin="anonymous"`